test/Microsoft.Health.Fhir.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -210,5 +210,30 @@ public async Task GivenResourceWithHistory_WhenSearchedWithParams_ThenOnlyCurren
             // Search for the given coding. The only thing returned should be the updated resource
             await ExecuteAndValidateBundle($"Observation?code={testCoding.System}|{testCoding.Code}", updatedResource);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenListOfResources_WhenSearchedWithCountSummary_ThenTotalCountShouldBeReturned()
+        {
+            const int numberOfResources = 5;
+
+            var tag = new Coding(string.Empty, Guid.NewGuid().ToString());
+
+            for (int i = 0; i < numberOfResources; i++)
+            {
+                Patient patient = Samples.GetDefaultPatient();
```

Comment:
Do we need to create the patient 5 times or can we just create 5 new patients using the same patient object?